.github/workflows/ant.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       run: luarocks install ar-io-ao-0.1-1.rockspec
 
     - name: Run Busted Tests
-      run: busted .
+      run: cd ant && busted .
```

Comment:
you can do `working-directory` at the top level to run actions against a specific folder

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunworking-directory